src/main/java/gui/BottomPane.java

```diff
@@ -0,0 +1,26 @@
+package gui;
+
+import javafx.geometry.Insets;
+import javafx.scene.control.Label;
+import javafx.scene.layout.HBox;
+
+/**
+ * Class that represents the whole of bottom-level elements in the gui.
+ * In other words, the bar at the bottom, at the least, goes here.
+ */
+class BottomPane extends HBox {
+
+    /**
+     * BottomPane Constructor.
+     */
+    BottomPane() {
+        // border style to mark it, for debugging for now.
+        setStyle("-fx-border-style: solid inside;"
+                + "-fx-border-width: 1;");
+        setPadding(new Insets(5, 10, 5, 10));
```

Comment:
I think this would be too much of a hassle for the gui. It should be quite easy to find everything when we put everything in the correct corresponding gui file/module.